R/utilities-dimensions.R

```diff
@@ -0,0 +1,160 @@
+#' Dimension existence
+#'
+#' @param dimension String name of the dimension.
+#' @details Returns \code{TRUE} if the provided dimension is supported otherwise \code{FALSE}
+#' @export
+hasDimension <- function(dimension) {

```

Comment:
@PavelBal I have renamed this to hasDimension and it is using the core method

Reply:
Why do you prefer the core method? We still have the enum Dimensions (or do you think we do not need it? I would disagree) which is created by calling `allAvailableDimensions`.

Reply:
Why is it called `HasDimension` btw? What does or does not have the dimension? :D @msevestre

Reply:
You are looking at this from the system point of view " Does OSPSUITE HAVE The dimension xxx"
so  `if(ospsuite::hasDimension("Mass"))` reads better than `if(ospsuite::existsDImension("Mass"))` in my opinion

But this is also because we had hasUnit. and not existsUnit. Naming is hard and consistency is even harder